data/json/furniture_and_terrain/furniture-medical.json

```diff
@@ -402,7 +402,7 @@
     "type": "furniture",
     "id": "f_HPLC",
     "name": "high performance liquid chromatographer",
-    "description": "This high-tech tool would, with electricity and an experienced user, be a very useful way to separate chemicals in a liquid or aqueous state, based on their affinity, into a solid state in a test tube.  At least, that's what the label says.",
+    "description": "This high-tech tool would, with electricity and an experienced user, be a very useful way to separate chemicals in a liquid or aqueous phase, based on their affinity, to a solid state in a tube.  At least, that's what the label says.",
```

Comment:
After some more thinking ¨based on their affinity, to the stationary phase in the tube." seems to me even better and 
scientifically more correct.